lib/Rx/Observable.php

```diff
@@ -1634,4 +1635,22 @@ public static function race(array $observables)
             return new RaceOperator();
         });
     }
+
+    /**
+     * Take only one element from sequence at specified index.
+     * First element in sequence have index 0.
+     *
+     * @param int $index
+     * @return AnonymousObservable
+     *
+     * @demo elementAt/elementAt.php
+     * @demo elementAt/elementAtOverflow.php
+     * @operator
+     * @reactivex elementAt
+     */
+    public function elementAt($index=null){
```

Comment:
A couple of things here:
1. Why is `$index` optional?
2. There should be an optional default value parameter
3. nit pick:  there should be spaces around `=`